### ISSUE TITLE
icon way to big

### ISSUE THREAD

**Issue by trebmuh** [splitbrain/dokuwiki-plugin-searchindex#50]
I've just update from Greebo to Hogfather and I enjoyed this to update this plugin.
It seems to me that #44 / #45 is still not fixed.

See ?do=admin :
![admin_searchindexmanager](https://user-images.githubusercontent.com/8705846/91915052-d8e1c100-ecb9-11ea-9e5a-0daf2b412c9e.png)

See ?do=admin&page=extension :
![version_searchindexmanager](https://user-images.githubusercontent.com/8705846/91915087-ed25be00-ecb9-11ea-99a9-2e013279cb23.png)

Note that I've tried to "_reinstall_" the plugin without more success.

**Comment by splitbrain** [comment]
have you actually read the issues you linked?

**Comment by trebmuh** [comment]
Yes, I've read it and it's not really clear if that has been solved or not (in fact, it looks like it hasn't since I reported here that it's still the same situation).

Of course, I could workaround it by adding a CSS rules as explained by @tmo26 [here](https://github.com/splitbrain/dokuwiki-plugin-searchindex/issues/44#issuecomment-639531083), or I could try to workaround it by applying manually #45 .

It seems to me that it should be fixed by a plugin update since this having a big icon as a "basic" situation isn't what is expected by a end-user. Maybe I'm not seeing clearly what the issue is and what you do recommand since I'm not a professional daily coder, just a end-user.

**Comment by trebmuh** [comment]
@splitbrain : yes, I've read it and it's not really clear if that has been solved or not (in fact, it looks like it hasn't since I reported here that it's still the same situation).

Of course, I could workaround it by adding a CSS rules as explained by @tmo26 [here](https://github.com/splitbrain/dokuwiki-plugin-searchindex/issues/44#issuecomment-639531083), or I could try to workaround it by applying manually #45 .

It seems to me that it should be fixed by a plugin update since this having a big icon as a "basic" situation isn't what is expected by a end-user. Maybe I'm not seeing clearly what the issue is and what you do recommand since I'm not a professional daily coder, just a end-user.

**Comment by tmo26** [comment]
The key sentence in #44 is:
> The size of admin icons should be set by the template.

If any icon in the admin area is too big, then that's a problem of the template, and not a problem of the plugin or dokuwiki itself.

**Comment by trebmuh** [comment]
_(I've applied @tmo26 's fix and that works fine)_

That said, there is something I don't understand. I can't remember touching manually this `lib/tpl/mytheme/css/_admin.css` file. Probably I've copied the whole `lib/tpl/mytheme/` years ago from a "basic theme" before tweaking it to the colors and stuff I wanted. I'd like to check if the issue is fixed in such a  "basic" theme is such a thing exists. Does it?
